packages/wrangler/src/pages/deploy.ts

```diff
@@ -454,6 +454,13 @@ ${failureMessage}`,
 		url: deploymentResponse.url,
 		alias,
 		environment: deploymentResponse.environment,
+		production_branch: deploymentResponse.production_branch,
+		deployment_trigger: {
+			metadata: {
+				commit_hash:
+					deploymentResponse.deployment_trigger?.metadata?.commit_hash ?? "",
```

Comment:
Under what circumstances would `deployment_trigger` ever be undefined?

Reply:
I'm not sure, it's not a required field in our openapi schema https://developers.cloudflare.com/api/operations/pages-deployment-create-deployment

Reply:
For Pages Direct Upload deployments, I believe the git ref is optional because you could deploy a Pages project that is not in a repo